app/schemas/com.ciandt.dragonfly.example.data.local.AppDatabase/2.json

```diff
@@ -0,0 +1,272 @@
+{
```

Comment:
What do you think about update the version of room? alpha8 produces a new schema because now they use NOT NULL. So, we can do 2 in 1 migration.